ethcontract/src/secret.rs

```diff
@@ -103,6 +103,10 @@ impl Key for &'_ PrivateKey {
     fn address(&self) -> Address {
         self.public_address()
     }
+
+    fn sign_message(&self, message: &[u8]) -> Result<Signature, SigningError> {
+        todo!()
```

Comment:
Ahh, is this is because the recovery value is 0 or 1 for the new transaction types instead 27 or 28? Wouldn't this be as easy as:
```suggestion
        let signature = self.sign(message, None)?;
        Ok(Signature { v: signature.v - 27, ..signature })
```

Reply:
Not 100% sure, but I expect this function to be erased when I fix the cargo dependencies. For now I leave this comment as unresolved.

Reply:
Implemented.